WD-Firmware/board/nexys_a7/bsp/env/freedom-e300-hifive1/platform.h

```diff
@@ -0,0 +1,133 @@
+// See LICENSE for license details.
```

Comment:
remove/rename the folder freedom-e300-hifive1

Reply:
[will be done after new main]

Reply:
Done by Ofer